### PR TITLE
Improve discreet mode

### LINKED ISSUE #677: Discreet mode is not clear
It's been at least 2 times that I recall that users had trouble realizing that they had discreet mode on and thought that something was wrong and didn't know how to fix it. Maybe we can find a better way on hiding the amounts (maybe add a tooltip that says "hidden due to discreet mode on" or sth.

### PR DESCRIPTION
**Pull request by mvaivre**
Hopefully fixes #677

### CODE REVIEW

File: src/components/Amount.tsx [hunk 1]
```diff
@@ -84,10 +89,14 @@ const Amount = ({
   const [integralPart, fractionalPart] = amount.split('.')
 
   return (
-    <span className={className} tabIndex={tabIndex ?? -1}>
-      {discreetMode ? (
-        '•••'
-      ) : value !== undefined ? (
+    <AmountStyled
+      {...{ className, color, value, highlight, tabIndex: tabIndex ?? -1, discreetMode }}
```
Comment (mvaivre) [inline src/components/Amount.tsx:93]: Note the spreading technique. This makes it DRYier when passing props down to the styled component. I've added this proposition to our Notion documentation. We're not forced to use this if you don't like it obviously!

**Review by nop33 — COMMENTED** [review]
I think it's much clearer now, good job 👍 

1. Shall we make this PR against `next`?
2. I found a missed "3-dot" element:
    image
3. In the Address Details modal, the percentage seems to be stack in loading mode. Not sure if it's related to these changes
    image

File: locales/de-DE/translation.json [hunk 2]
```diff
@@ -421,5 +422,5 @@
   "Your addresses": "Ihre Adressen",
   "Your CSV file has been generated successfully.": "Your CSV file has been generated successfully.",
   "Your CSV file is being compiled in the background.": "Your CSV file is being compiled in the background.",
-  "Your Wallet": "Ihre Wallet"
+  "Your Wallet": "Your Wallet"
```
Comment (nop33) [inline locales/de-DE/translation.json:425]: This translation was actually correct 😅

File: locales/id-ID/translation.json [hunk 3]
```diff
@@ -421,5 +422,5 @@
   "Your addresses": "Your addresses",
   "Your CSV file has been generated successfully.": "Your CSV file has been generated successfully.",
   "Your CSV file is being compiled in the background.": "Your CSV file is being compiled in the background.",
-  "Your Wallet": "Dompet Anda"
+  "Your Wallet": "Your Wallet"
```
Comment (nop33) [inline locales/id-ID/translation.json:425]: Same here

File: locales/es-ES/translation.json [hunk 4]
```diff
@@ -421,5 +422,5 @@
   "Your addresses": "Your addresses",
   "Your CSV file has been generated successfully.": "Your CSV file has been generated successfully.",
   "Your CSV file is being compiled in the background.": "Your CSV file is being compiled in the background.",
-  "Your Wallet": "Su billetera"
+  "Your Wallet": "Your Wallet"
```
Comment (nop33) [inline locales/es-ES/translation.json:425]: I guess all that was needed was to capitalize the word `billetera`...

File: locales/pt-PT/translation.json [hunk 5]
```diff
@@ -421,5 +422,5 @@
   "Your addresses": "Your addresses",
   "Your CSV file has been generated successfully.": "Your CSV file has been generated successfully.",
   "Your CSV file is being compiled in the background.": "Your CSV file is being compiled in the background.",
-  "Your Wallet": "A sua carteira"
+  "Your Wallet": "Your Wallet"
```
Comment (nop33) [inline locales/pt-PT/translation.json:425]: Not sure how to capitalize this one tho, so I'd leave it like this.

**Pull request merged**